test/mocha/release.js

```diff
@@ -22,10 +22,9 @@ describe("test/benchmark.js", function() {
         "-b",
         "-b bracketize",
         "-m",
-        "-mc passes=3",
-        "-mc passes=3,toplevel",
-        "-mc passes=3,unsafe",
-        "-mc keep_fargs=false,passes=3",
```

Comment:
I know. I thought total CPU limit across all travis jobs had some bearing on the timeouts.